_inc/client/traffic/related-posts.jsx

```diff
@@ -63,12 +63,19 @@ class RelatedPostsComponent extends React.Component {
 					disableInDevMode
 					module={ this.props.getModule( 'related-posts' ) }
 					support={ {
+						text: __(
+							'The related posts feature scans all of your posts’ contents, ' +
```

Comment:
Should it be "content", since we say "it" afterwards?

```suggestion
							'The related posts feature scans all of your posts’ content, ' +
```